### PR TITLE
Convert twitter to radix

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

File: twitter_radix/twitter_radix/state/base.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    password: str = Field()
+
+
+class Tweet(rx.Model, table=True):
+    """A table of Tweets."""
+
+    content: str = Field()
+    created_at: str = Field()
+
+    author: str = Field()
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/state/base.py:32]: I would move them into a separate file. These are not all part of a state, this is the DB data model

File: twitter_radix/twitter_radix/state/base.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    """
+
+    followed_username: str = Field(primary_key=True)
+    follower_username: str = Field(primary_key=True)
+
+
+class User(rx.Model, table=True):
+    """A table of Users."""
+
+    username: str = Field()
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/state/base.py:22]: we don't need the `= Field()` here for default column definition, let's remove them if the old app had it.

File: twitter_radix/twitter_radix/components/container.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+def container(*children, **props):
+    """A fixed container based on a 960px grid."""
+    # Enable override of default props.
+    props = (
+        dict(
+            width="100%",
+            max_width="960px",
+            background="white",
+            height="100%",
+            #px=[4, 12],
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/components/container.py:13]: can we remove the commented code. there also seems to be an empty line below line 15

File: twitter_radix/twitter_radix/layouts/auth.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+from ..components import container
+
+
+def auth_layout(*args):
+    """The shared layout for the login and sign up pages."""
+    return rx.box(
+        container(
+            rx.heading(
+                rx.chakra.span("Welcome to PySocial!"),
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/layouts/auth.py:12]: what's the replacement in 0.4.0 for this?

File: twitter_radix/twitter_radix/pages/home.py [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+                    padding="1em",
+                ),
+            ),
+            padding="2rem",
+            border_radius="0.5rem",
+            border="1px solid #eaeaea",
+            width="100%",
+        ),
+        align_items="start",
+        gap="4",
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/pages/home.py:99]: gap => spacing? there's other occurrences in this PR too

File: twitter_radix/twitter_radix/pages/home.py [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    return rx.grid(
+        rx.vstack(
+            rx.avatar(fallback=State.user.username.to_string(), size="4"),
+            padding="1.5rem",
+        ),
+        rx.box(
+            rx.flex(
+                rx.text_area(
+                    width="100%",
+                    #border="none",
```
Comment (martinxu9) [inline twitter_radix/twitter_radix/pages/home.py:128]: remove commented code

File: twitter_radix/requirements.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+reflex>=0.3.7
```
Comment (martinxu9) [inline twitter_radix/requirements.txt:1]: 0.4.0 now?

**Review by martinxu9 — CHANGES_REQUESTED** [review]

**Comment by martinxu9** [comment]
Also I'm moving the old apps into a subfolder, let's keep the name as it is: `twitter` not `twitter_radix`

File: twitter_radix/twitter_radix/state/base.py (hunk 1 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/state/base.py:32]: moved

File: twitter_radix/twitter_radix/state/base.py (hunk 2 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/state/base.py:22]: done

File: twitter_radix/twitter_radix/components/container.py (hunk 3 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/components/container.py:13]: removed

File: twitter_radix/twitter_radix/layouts/auth.py (hunk 4 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/layouts/auth.py:12]: removed

File: twitter_radix/twitter_radix/pages/home.py (hunk 6 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/pages/home.py:128]: removed

File: twitter_radix/twitter_radix/pages/home.py (hunk 5 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/pages/home.py:99]: changed

File: twitter_radix/twitter_radix/layouts/auth.py (hunk 4 above)
Comment (tgberkeley) [reply twitter_radix/twitter_radix/layouts/auth.py:12]: changed to text(..., as_"span")

**Pull request closed without merging**